app/controllers/course/user_invitations_controller.rb

```diff
@@ -199,11 +200,15 @@ def invalid_invitations
   end
 
   # Returns the successful invitation creation message based on file or entry invitation.
-  def create_success_message
+  def create_success_message(new_invitations, duplicate_invitations, new_course_users, duplicate_course_users)
```

Comment:
Maybe we use `existing_invitations`? So we don't confuse this with the duplication features. Same for `course_users`.